itest/opstackl2/op_test_manager.go

```diff
@@ -328,39 +327,17 @@ func (ctm *OpL2ConsumerTestManager) WaitForTargetBlockPubRand(t *testing.T, fpLi
 	return targetBlockHeight
 }
 
-func (ctm *OpL2ConsumerTestManager) RegisterBBNFinalityProvider(t *testing.T) *btcec.PublicKey {
+// can be used for both the Babylon and Consumer FPs
+func (ctm *OpL2ConsumerTestManager) RegisterFinalityProvider(t *testing.T, chainId string, n int) []*bbntypes.BIP340PubKey {
```

Comment:
```suggestion
func (ctm *OpL2ConsumerTestManager) RegisterFinalityProvider(t *testing.T, consumerID string, n int) []*bbntypes.BIP340PubKey {
```

Let's use `consumerID` followed by Babylon's terminology